### ISSUE TITLE
Evaluation on SPIn-NeRF dataset

### ISSUE THREAD

**Issue by XuHu0529** [Jumpat/SegmentAnythingin3D#34]
Hi, thank you for your amazing work. 

I wonder how many views are used to train the model and do the evaluation? I have read the SPIn-NeRF paper, but I didn't find it. May I have overlooked something. Could u please tell me or give me some instructions on how to find it?

Thanks a lot!!!

**Comment by Jumpat** [comment]
The SPIn-NeRF dataset is based on some standard NeRF dataset like LLFF. To train the NeRF model we utilize all images (the number may be about 20) provided by the dataset for each scene. 

Strictly speaking, for SA3D, there is no such process as training since there is no ground truth used in the generation of the mask grids. In practice we simply run SA3D on all images provided by the dataset.

**Comment by XuHu0529** [comment]
Get it! So the mIoU and Acc are also calculated on all images? By the way, is the evaluation of MVSeg also on the same setting?

Thank you for your reply.

**Comment by Jumpat** [comment]
You're welcome. The author of SPIn-NeRF (MVSeg) provided the annotations on those datasets. Not all images in the datasets are labelled so we only evaluate the method on those images with annotation. 

We follow their setting to conduct the so-called label propagation (input a reference view with its corresponding gt mask and calculate the IoU on all given annotated  images)

**Comment by XuHu0529** [comment]
OK, got it. Thank you very much!

**Comment by namepllet** [comment]
Thank you for your amazing work.

Could you explain what reference view is utilized for each scene ?

**Comment by Jumpat** [comment]
Hello! For the SPIn-NeRF dataset, the reference view is set as the first view in the training set.
